flint/flint-spark-integration/src/main/scala/org/apache/spark/sql/flint/FlintDataSourceV2.scala

```diff
@@ -3,35 +3,42 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-package org.apache.spark.sql.v2
+package org.apache.spark.sql.flint
```

Comment:
Is this convention that all Spark data source code should be in `org.apache.spark.sql` package?

Reply:
We need to use [private] class defined in org.apache.spark.sql. In case it is spark integration, the answer is yes.

Reply:
I see. Thanks!